backend/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM python:3.11.1-alpine3.16
+FROM python:3.11-alpine3.16
 
 # Set work dir for relative paths
-RUN mkdir -p /app
 WORKDIR /app
```

Comment:
The `WORKDIR` here changes the working directory to `/app` so the `COPY` command below is already operating from within `/app`. Changing the destination to `.` then does the trick of copying the file into `/app`.